src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/DBCaseRepository.java

```diff
@@ -209,6 +201,7 @@ public Claim saveClaim(User user, Claim claim) {
 
         return claimRepository
             .getClaimByExternalId(claim.getExternalId())
+            .map(fetchedClaim -> fetchedClaim.toBuilder().id(null).build())
```

Comment:
follow other approach.

Reply:
We can consider the option to add the property in our `Claim` object to distinguish the origin of the data store(CCD or CLAIM_STORE), in order to identify this in our front end. We don't need to store this in database/CCD. The relevant repository class can populate it accordingly.

Reply:
@dharmendrak , @sabahirfan we shouldnt create additional property in claim to distinguish if its CCD or Claimstore. I dont know if the previousServiceReferenceNumber will be useful? Let we have a discussion

Reply:
`previousServiceReferenceNumber` will not work as that is not in CMC domain model.

Reply:
@sabahirfan I think the previousServiceReferenceNumber is mapped to referenceNumber rt? I think you did that too.